### PR TITLE
Update sdk initialization

### PR DESCRIPTION
**Pull request by dignacio0815**

### CODE REVIEW

File: src/scripts/client.js [hunk 1]
```diff
@@ -45,7 +45,11 @@ const main = async () => {
     // WARNING: It is best practice to use the VoxeetSDK.initializeToken function to initialize the SDK.
     // Please read the documentation at:
     // https://docs.dolby.io/communications-apis/docs/initializing-javascript
-    VoxeetSDK.initialize('customerKey', 'customerSecret');
+    VoxeetSDK.initializeToken(access_token, () => {
+      // This callback is called when the token needs to be refreshed
```
Comment (FabienLavocat) [inline src/scripts/client.js]: Return a promise with the same access_token

File: src/scripts/client.js [hunk 2]
```diff
@@ -45,7 +45,11 @@ const main = async () => {
     // WARNING: It is best practice to use the VoxeetSDK.initializeToken function to initialize the SDK.
```
Comment (FabienLavocat) [inline src/scripts/client.js]: This warning is not necessary anymore. Just keep the URL below.

File: src/scripts/client.js [hunk 3]
```diff
@@ -45,7 +45,11 @@ const main = async () => {
...
     // Please read the documentation at:
     // https://docs.dolby.io/communications-apis/docs/initializing-javascript
-    VoxeetSDK.initialize('customerKey', 'customerSecret');
+    VoxeetSDK.initializeToken(access_token, () => {
+      // This callback is called when the token needs to be refreshed
+    }).catch(error => {
```
Comment (FabienLavocat) [inline src/scripts/client.js]: This is not an async function, you can't catch anything.

File: src/scripts/client.js (hunk 1 above)
Comment (FabienLavocat) [reply src/scripts/client.js]: Where is `access_token` defined?

**Review by FabienLavocat — CHANGES_REQUESTED** [review]

File: src/scripts/client.js (hunk 3 above)
Comment (dignacio0815) [reply src/scripts/client.js]: Sounds good, will update this. We need to update this in the docs as well as it references a .catch(error)

**Review by FabienLavocat — APPROVED** [review]

File: src/scripts/client.js [hunk 4]
```diff
@@ -42,10 +42,15 @@ const main = async () => {
 
   try {
     // Initialize the Voxeet SDK
-    // WARNING: It is best practice to use the VoxeetSDK.initializeToken function to initialize the SDK.
     // Please read the documentation at:
     // https://docs.dolby.io/communications-apis/docs/initializing-javascript
-    VoxeetSDK.initialize('customerKey', 'customerSecret');
+    // Grab your demos client token from your dashboard and insert into access_token
```
Comment (dzfill) [inline src/scripts/client.js]: Update to    // Generate a test client access token from the Dolby.io dashboard and insert into access_token variable

File: src/scripts/client.js [hunk 5]
```diff
@@ -42,10 +42,15 @@ const main = async () => {
...
+    let access_token = "<INSERT DEMOS CLIENT ACCESS TOKEN HERE>";
```
Comment (dzfill) [inline src/scripts/client.js]: Change INSERT DEMOS CLIENT ACCESS TOKEN HERE to TestClientAccessToken

**Pull request merged**